community/kernel/src/main/java/org/neo4j/helpers/collection/Visitable.java

```diff
@@ -19,7 +19,7 @@
  */
 package org.neo4j.helpers.collection;
 
-public interface Visitable<T>
+public interface Visitable<V>
 {
-    void accept(Visitor<T, RuntimeException> visitor);
+    void accept( V visitor );
```

Comment:
I think the fact that no one is using `Visitable` is actually a problem which we are no making any better by changing the `Visitable` abstraction.

As I see this is that `Visitor` and `Visitable` are tightly together and they define a kind of contract on how the visitor pattern should work. Making that generalization somehow breaks that which looks wrong to me.